activerecord/CHANGELOG.md

```diff
@@ -1,3 +1,18 @@
+*   Introduce ApplicationRecord, an Active Record layer super type.
```

Comment:
A "Active Record layer super type" = ??? to most people.

`ApplicationRecord` is a new superclass for all app models, analogous to app controllers subclassing `ApplicationController` instead of `ActionController::Base`. This gives apps a single spot to configure app-wide model behavior.